robot/Cumulus/resources/NPSP.py

```diff
@@ -1200,7 +1199,7 @@ def verify_record_is_created_in_database(self,object_name,id):
         """Verifies that a record with specified id is saved 
            in specified object table in database and returns the record"""
         record=self.salesforce.salesforce_get(object_name,id)
-        self.builtin.should_not_be_empty(record)
+        self.builtin.should_not_be_empty(record, msg="the database object {} with id {} is not in the database".format(object_name,id))
```

Comment:
A very trivial change. Consider capitalizing the starting letter of a sentence?

Reply:
done